deploy/kubernetes/base/node.yaml

```diff
@@ -95,3 +97,12 @@ spec:
           hostPath:
             path: /sys
             type: Directory
+      tolerations:
+      # Blanket tolerations to make sure the node component is scheduled on all
+      # nodes.
+      - effect: NoSchedule
+        operator: Exists
+      - effect: NoExecute
+        operator: Exists
+      - key: CriticalAddonsOnly
```

Comment:
double check we can actually set things like this without being in the kube-system namespace.

Also, is system priorityclass useful?

Reply:
I wonder why we don't just to a blanket toleration like,
```
- operator: Exists
```

which matches all keys and effects?

Reply:
@misterikkit do you mean if we don't specify a `Key` the behavior will be to match *all* keys not *none* keys? (I haven't looked at docs but that feels backwards to me).

IIUC priority class will cause the driver to be ranked higher in scheduling order and reduce possibility of pre-emption (only by higher priority pods)

Reply:
found the doc for toleration. Updated